types/stripe/stripe-tests.ts

```diff
@@ -1018,6 +1018,10 @@ stripe.invoices.list({ customer: "cus_5rfJKDJkuxzh5Q", limit: 3 }).then(function
     // asynchronously called
 });
 
+stripe.invoices.retrieve("in_15fvyXEe31JkLCeQH7QbgZZb", { expand: ["subscription"] }).then(function (invoice) {
```

Comment:
Not necessarily for this PR, but it would be nice if the return type could vary depending on what value is passed to `expand`